devtools/__main__.py

```diff
@@ -64,7 +64,9 @@ def install() -> int:
     print(f'Found path `{install_path}` to install devtools into `builtins`')
     print('To install devtools, run the following command:\n')
     print(f'    python -m devtools print-code >> {install_path}\n')
-    if not install_path.is_relative_to(Path.home()):
+    try:
+        install_path.relative_to(Path.home())
+    except ValueError:
```

Comment:
you still need 

```py
    is_relative_to = install_path.relative_to(Path.home())
except ValueError:
    if not is_relative_to:
```

Reply:
`relative_to` will raise if the path isn't relative; if it doesn't raise, it's impossible the path isn't relative.

https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to

Reply:
For all versions, what about what we had before?

Reply:
I'm sorry, could you elaborate on your question a little more please?

Reply:
Oh ignore me, the method has changed.